README.md

```diff
@@ -5,52 +5,48 @@
 [![Code Climate](https://codeclimate.com/github/elwayman02/ember-sinon-qunit/badges/gpa.svg)](https://codeclimate.com/github/elwayman02/ember-sinon-qunit)
 [![Codacy Badge](https://api.codacy.com/project/badge/8c6fbb028801423fbd4b1bfe17c9b1a0)](https://www.codacy.com/app/hawker-jordan/ember-sinon-qunit)
 
-This addon integrates [`sinon`](http://jhawk.co/sinonjs) & [`ember-qunit`](http://jhawk.co/ember-qunit) 
-via [`ember-sinon`](http://jhawk.co/ember-sinon), originally inspired by [`sinon-qunit`](http://jhawk.co/sinon-qunit).
+This addon integrates [`sinon`](http://jhawk.co/sinonjs) & [`ember-qunit`](http://jhawk.co/ember-qunit).
 
-Why not simply use `ember-sinon` alone? Two reasons:
+Why not simply use `sinon` alone?
 
-1. `ember-sinon` does not handle cleanup of `ember-qunit` tests. While `sinon` 
-[sandboxes itself](https://sinonjs.org/guides/migrating-to-5.0), it's up to the user to 
-consistently clean up `sinon` after each test. `ember-sinon-qunit` automatically 
+`sinon` does not handle cleanup of `ember-qunit` tests. While `sinon`
+[sandboxes itself](https://sinonjs.org/guides/migrating-to-5.0), it's up to the user to
+consistently clean up `sinon` after each test. `ember-sinon-qunit` automatically
 restores `sinon`'s state to ensure nothing is leaked between tests. All spies/stubs created
 will be automatically restored to their original methods at the end of each test.
-2. `sinon` is a framework-agnostic library; as such, `ember-sinon` should be as well. This addon exists to enable
-`ember-sinon` to remove its `qunit` specific functionality, making it easier to utilize `ember-sinon` 
-with other addons like [`ember-cli-mocha`](http://jhawk.co/ember-cli-mocha), for example.
-
 
 ## Compatibility
 
-* Sinon.js v5.0.0 or above
-* Ember.js v3.28 or above
-* Ember CLI v3.28 or above
-* Node.js v14 or above
-
+- Sinon.js v5.0.0 or above
```

Comment:
```suggestion
- Sinon.js v15.0.0 or above
```

You mentioned in this PR that v15 is required now, right?

Reply:
Oh, right, good catch!